servo-tidy.toml

```diff
@@ -4,7 +4,7 @@ skip-check-licenses = false
 
 [ignore]
 # Ignored packages with duplicated versions
-packages = ["lazy_static"]
+packages = ["bitflags", "lazy_static"]
```

Comment:
There's no reasoning anywhere for why a dupe bitflags is required. I think that should be explained somewhere or a bug filed and referenced before we allow this.